test/e2e/dra/dra.go

```diff
@@ -2244,276 +2242,6 @@ var _ = framework.SIGDescribe("node")(framework.WithLabel("DRA"), func() {
 			mustFailToDelete(fictionalNodeClient, "fictional plugin", createdClusterSlice, matchVAPDeniedError(fictionalNodeName, createdClusterSlice))
 			mustDelete(f.ClientSet, "admin", createdClusterSlice)
 		})
-
-		f.It("controller manager metrics track ResourceClaim operations with correct labels", f.WithSerial(), f.WithFeatureGate(features.DRAAdminAccess), func(ctx context.Context) {
```

Comment:
merge issue? This seems unrelated

Reply:
Whoa, weird.  Let me take a look.

Reply:
I must have hit a wrong key during the rebase.  This is fixed.